openlr/router.cpp

```diff
@@ -168,8 +168,7 @@ Router::Edge Router::Edge::MakeNormal(Vertex const & u, Vertex const & v,
 // static
 Router::Edge Router::Edge::MakeSpecial(Vertex const & u, Vertex const & v)
 {
-  return Edge(u, v, routing::Edge::MakeFake(u.m_junction, v.m_junction, false /* partOfReal */),
-              true /* isSpecial */);
+  return Edge(u, v, routing::Edge::MakeFake(u.m_junction, v.m_junction), true /* isSpecial */);
```

Comment:
I cannot find a ctor which is used here. Can you compile this code?

Reply:
This is not routing::Edge. This one is openlr::Edge

Reply:
And yes, I can compile everything: make -j 8 all.

Reply:
Ok